hack/dev/Dockerfile.okteto

```diff
@@ -169,4 +157,6 @@ RUN cd /tmp && curl -fsSL -o helm.tar.gz "${HELM3_URL}" \
 #  && mv s3cmd-${S3CMD_VERSION}/S3 /usr/local/bin/S3 \
 #  && rm -rf s3cmd.tar.gz s3cmd.tar.gz.asc s3cmd-${S3CMD_VERSION}
 
+RUN make kotsadm
```

Comment:
Can we change the make commands to 
```
make build/run/debug
```
to keep it a standard over the projects?

Reply:
Sadly kots and kotsadm both share this make file so it creates a bit of a naming issue.   I want to get these standardized but I didn't want to go to deep on this PR right now